src/Backend/Core/Js/backend.js

```diff
@@ -123,6 +123,7 @@ jsBackend.navigation =
         jsBackend.navigation.mobile();
         jsBackend.navigation.toggleCollapse();
         jsBackend.navigation.tooltip();
+        jsBackend.navigation.modal();
```

Comment:
Meuh, wouldn't it be more useful to use a separate action and template? So it can be optimised for usage in a modal?

Reply:
I ended up changing this. Now it is using a dedicated template.